config/tests/samples/create/samples.go

```diff
@@ -358,6 +359,7 @@ func LoadSample(t *testing.T, sampleKey SampleKey, project testgcp.GCPProject) S
 type SampleKey struct {
 	Name      string
 	SourceDir string
+	APIGroup  string
```

Comment:
Nit: Would be good to add a comment about what this is, and maybe just export the GVK (because 1. it is strongly-typed and 2. you have it anyway).  But not a blocker.